docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs

```diff
@@ -10,39 +10,49 @@ public static class LbfgsMaximumEntropy
     {
         public static void Example()
         {
-            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
-            // as a catalog of available operations and as the source of randomness.
-            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            // Create a new context for ML.NET operations. It can be used for
+            // exception tracking and logging, as a catalog of available operations
+            // and as the source of randomness. Setting the seed to a fixed number
+            // in this example to make outputs deterministic.
             var mlContext = new MLContext(seed: 0);
 
             // Create a list of training data points.
             var dataPoints = GenerateRandomDataPoints(1000);
 
-            // Convert the list of data points to an IDataView object, which is consumable by ML.NET API.
+            // Convert the list of data points to an IDataView object, which is
+            // consumable by ML.NET API.
             var trainingData = mlContext.Data.LoadFromEnumerable(dataPoints);
 
             // Define the trainer.
             var pipeline =
-                    // Convert the string labels into key types.
-                    mlContext.Transforms.Conversion.MapValueToKey(nameof(DataPoint.Label))
-                    // Apply LbfgsMaximumEntropy multiclass trainer.
-                    .Append(mlContext.MulticlassClassification.Trainers.LbfgsMaximumEntropy());
+            // Convert the string labels into key types.
+                mlContext.Transforms.Conversion
+                .MapValueToKey(nameof(DataPoint.Label))
+            // Apply LbfgsMaximumEntropy multiclass trainer.
+                .Append(mlContext.MulticlassClassification.Trainers
+                .LbfgsMaximumEntropy());
+
 
             // Train the model.
             var model = pipeline.Fit(trainingData);
 
-            // Create testing data. Use different random seed to make it different from training data.
-            var testData = mlContext.Data.LoadFromEnumerable(GenerateRandomDataPoints(500, seed: 123));
+            // Create testing data. Use different random seed to make it different
+            // from training data.
+            var testData = mlContext.Data
+                .LoadFromEnumerable(GenerateRandomDataPoints(500, seed: 123));
 
             // Run the model on test data set.
             var transformedTestData = model.Transform(testData);
 
             // Convert IDataView object to a list.
-            var predictions = mlContext.Data.CreateEnumerable<Prediction>(transformedTestData, reuseRowObject: false).ToList();
+            var predictions = mlContext.Data
+                .CreateEnumerable<Prediction>(transformedTestData,
+                reuseRowObject: false).ToList();
```

Comment:
📝 This is difficult to read. See preferred form in https://github.com/dotnet/machinelearning/pull/3941#discussion_r299511862